upgrading/rosa-upgrading-sts.adoc

```diff
@@ -1,31 +1,41 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="rosa-upgrading-sts"]
-= Upgrading ROSA Classic clusters
+= Upgrading ROSA (classic architecture) clusters
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-upgrading-sts
 
 toc::[]
 
+Use one of the following methods to upgrade ROSA (classic architecture) clusters:
+
+* Manually through the ROSA CLI (`rosa`) - Start a one-time immediate upgrade or schedule a one-time upgrade for a future date or time.
+* Manually through the {cluster-manager} UI - Start a one-time immediate upgrade or schedule a one-time upgrade for a future date or time; or schedule an upgrade window for automatic recurring upgrades whenever a new z-version is available.
+
 [id="rosa-lifecycle-policy_{context}"]
 == Life cycle policies and planning
 
 To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information and life cycle dates.
 
-Upgrades are manually initiated or automatically scheduled. Red{nbsp}Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.
+You can use update channels to decide which Red Hat OpenShift Container Platform minor version to update your clusters to. {product-title} supports updates only through the `stable` channel. To learn more about OpenShift update channels and releases, see link:https://docs.openshift.com/container-platform/4.15/updating/understanding_updates/understanding-update-channels-release.html[Understanding update channels and releases].
```

Comment:
Hi, I guess this is a hard link maybe you can't use an xref since this is the ROSA doc set - but shouldn't it be linked to 4.16 since that is the latest version, if you have to use a hard link?

Reply:
Changed to "You can use update channels to decide which Red Hat OpenShift Container Platform minor version to update your clusters to. {product-title} supports updates only through the `stable` channel. To learn more about OpenShift update channels and releases, see link:https://docs.openshift.com/container-platform/latest/updating/understanding_updates/understanding-update-channels-release.html[Understanding update channels and releases]."